test/smoke/openshift.go

```diff
@@ -133,3 +156,34 @@ func waitAndRetrieveRouteURI(namespace, serviceName string) (string, error) {
 	GetLogger(namespace).Debugf("Got route %s\n", routeURI)
 	return routeURI, nil
 }
+
+// waitForPods waits for pods with specific label to be available and running
+func waitForPods(namespace, labelName, labelValue string, numberOfPods, timeoutInMin int) error {
```

Comment:
Since those functions are related to `Pods` resource, why not keep them on `kubernetes.go` instead?

Reply:
good point, can do that in next PR, can move there also route functions

Reply:
Fixed in this PR.